lib/person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Kindly ensure that `Getters` and `Setters` for `:rentals` are present. Currently, you only have `getters` set for `:rentals` so kindly fix that :wink: